presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -959,6 +959,114 @@ where :math:`f(x)` is the partial density function of :math:`x`.
     The function uses the stream summary data structure proposed in the paper
     `Efficient computation of frequent and top-k elements in data streams <https://www.cse.ust.hk/~raywong/comp5331/References/EfficientComputationOfFrequentAndTop-kElementsInDataStreams.pdf>`_ by A.Metwally, D.Agrawal and A.Abbadi.
 
+Reservoir Sample Functions
+-------------------------------
+
+Reservoir sample functions use a fixed sample size, as opposed to
+:ref:`TABLESAMPLE <sql-tablesample>`. Fixed sample sizes always result in a
+fixed total size while still guaranteeing that each record in dataset has an
+equal probability of being chosen. See [Vitter1985]_
+
+.. function:: reservoir_sample(initial_sample: array(T), initial_processed_count: bigint, values_to_sample: T, desired_sample_size: int) -> row(processed_count: bigint, sample: array(T))
+
+    Computes a new reservoir sample given:
+    
+    - ``initial_sample``: an initial sample array, or ``NULL`` if creating a new sample
+    - ``initial_processed_count``: the number of records processed to generate the initial sample array.
+      This should be 0 or ``NULL`` if ``initital_sample`` is ``NULL``.
+    - ``values_to_sample``: the column to sample from
+    - ``desired_sample_size``: the size of reservoir sample
+
+    The function outputs a single row type with two columns:
+
+    #. The processed count. This is the total number of rows the
```

Comment:
```suggestion
    #. Processed count: the total number of rows the
```
Very minor suggestion to change these two items to be consistent with your writing style in the preceding unordered list.

Reply:
fixed